paulshryock/paul-shryock
# [#9/#10]: Optimize and minify SVG

## Summary
SVG files should be optimized, and minified in production.

## Testing
- Run `gulp svg`
- See that SVG files are optimized, minified in production, and output to `/build`.

## Issue(s)

- Closes #9
- Closes #10

## Changelog

### Added
- Optimize SVG. [#9]
- Minify SVG in production. [#10]

## Release Checklist
